cdk/lib/bastion.ts

```diff
@@ -7,31 +7,27 @@ const DEFAULT_AMI_MAP = {
 
 export interface CacclSshBastionProps {
```

Comment:
Great to see that you're using typescript as well :) I'm wondering if we might want to compare notes on style and general practices.

Reply:
love typescript. i worry that i'm cargo-culting some lot of the configuration because i'm usually starting with whatever scaffolding the cdk generates when I start a project. Would definitely be interested in your starting project setup.